ls1intum/artemis-ios
# `Communication`: Support File Sharing

### Problem Description
This PR introduces a new feature that allows users to upload files directly to a conversation.

### Changes
 
**File Upload UI:**

Added buttons for file in message area

**Validation:**

Enforced a 5MB file size limit.
Restricted uploads to supported file types.

### Steps for testing
Test the new buttons added to the messsage field for file upload.
Upload a file or image and verify:
Files/images are displayed with appropriate markdown.

### Screenshots

<img width="370" alt="Screenshot 2024-12-14 at 00 32 17" src="https://github.com/user-attachments/assets/ac94f92c-6662-48bb-b6c1-e9d719a61b92" />
